src/main/java/bio/overture/rollcall/repository/IndexRepository.java

```diff
@@ -76,27 +77,35 @@ public IndexRepository(@NonNull RestHighLevelClient client) {
 
   @SneakyThrows
   public boolean removeAlias(@NonNull String alias, @NonNull List<String> indices) {
+    return aliasActions(alias, Collections.emptyList(), indices);
+  }
+
+  @SneakyThrows
+  public boolean addAlias(@NonNull String alias, @NonNull List<String> indices) {
+    return aliasActions(alias, indices, Collections.emptyList());
+  }
+
+  @SneakyThrows
+  public boolean aliasActions(@NonNull String alias, @NonNull List<String> indicesToAddToAlias, @NonNull List<String> indicesToRemoveFromAlias) {
```

Comment:
All the code is fine but this method name is confusing. I wouldn't know to look for this and would probably run removeAlias then addAlias separately